cmd/notation/internal/display/metadata/tree/inspect.go

```diff
@@ -0,0 +1,191 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tree
+
+import (
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"fmt"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-go/plugin/proto"
+	"github.com/notaryproject/notation-go/registry"
+	"github.com/notaryproject/notation/cmd/notation/internal/output"
+	"github.com/notaryproject/notation/internal/envelope"
+	"github.com/notaryproject/notation/internal/tree"
+	"github.com/notaryproject/tspclient-go"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// InspectHandler is a handler for inspecting metadata information and rendering
+// it in a tree format.
+type InspectHandler struct {
+	printer *output.Printer
+
+	root        *tree.Node
+	cncfSigNode *tree.Node
```

Comment:
The field name `cncfSigNode` is confusing. Let's consider a better name.

Reply:
Updated to be `notationSignaturesNode`